front/README.md

```diff
@@ -84,35 +84,104 @@ Afterwards:
 
 ## Testing
```

Comment:
Please update this section to mention that we are slowly deprecating React Testing Library and move slowly to playwright. So, keep the old instructions in.
Also mention that all new test should be written in playwright.